editor/src/components/canvas/canvas-wrapper-component.tsx

```diff
@@ -73,11 +81,117 @@ export const CanvasWrapperComponent = betterReactMemo(
           />
         ) : null}
         {safeMode ? <SafeModeErrorOverlay /> : <ErrorOverlayComponent />}
+        <ModeSelectButtons />
       </FlexColumn>
     )
   },
 )
 
+interface ModeSelectButtonProps {
+  selected: boolean
+  title: string
+  onMouseDown: () => void
+}
+
+const ModeSelectButton = betterReactMemo('ModeSelectButton', (props: ModeSelectButtonProps) => {
+  return props.selected ? (
+    <div
+      // this is the selected variant. No hover effects on this one
+      style={{
+        padding: '2px 4px',
+        background: colorTheme.emphasizedBackground.value,
```

Comment:
I think this wants `secondary`, since the background of the parent is `emphasized` - right now you don't see the selected orb. Same with the hover effects in the one below.

(default editor bg is 98% lightness, `colorTheme.emphasized` is 100% (i.e white), and `colorTheme.secondary` etc are darker at 96% lightness etc. Which, of course, is also a form of emphasis :facepalm:. )

Reply:
Fixed!